src/Todo.tsx

```diff
@@ -1,10 +1,42 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import CreateList from './CreateList';
+
+export async function GetTodoList() {
+    const response = await fetch('http://127.0.0.1:8000/todo-list/')
+
+    if (!response.ok){
```

Comment:
When you get some way to stub out the requests, you should be able to test this part -

Reply:
Agreed.